tasks/create_config.yml

```diff
@@ -126,30 +137,20 @@
           # *CS TODO: Add the SSL support to mesh mode like node_base from the puppet (internal_tls), see https://github.com/openstack/puppet-tripleo/blob/432b0497f7d0d051f2b4bf34c3e841f2b97bbb44/manifests/profile/base/metrics/qdr.pp#L141
           connectors: "{{ connectors + [{
                         'host': item,
-                        'port': '5668',
+                        'port': '5667',
                         'role': 'inter-router',
                         'verifyHostname': 'no',
                         'saslMechanisms': 'ANONYMOUS'}]
                       }}"
 # End of mesh-mode block
 
-- name: Augment ssl_profiles with internal profile if supplied
-  set_fact:
-    ssl_profiles: "{{ ssl_profiles +
-                     [{ 'name': 'tlsProfile',
-                        'certFile': listener_ssl_cert_file,
-                        'keyFile': listener_ssl_key_file,
-                        'caCertFile': tripleo_internal_tlscafile
-                      }] }}"
-  when: tripleo_enable_internal_tls
-
 - name: "Include QDR config role"
   block:
     - name: "Generate the qdr config files"
       include_role:
         name: qdr_config
       vars:
-        # These vars are passed through unchanged
+        # These vars are passed through from tripleo unchanged
```

Comment:
I should have another PR coming soom that renames this to keep the qdr_ prefix in the underlying qdr_config role, eliminating this block